lib/Twig/Cache/Filesystem.php

```diff
@@ -98,6 +90,6 @@ public function write($key, $content)
      */
     public function getTimestamp($key)
     {
-        return filemtime($key);
+        return @filemtime($key);
```

Comment:
Noted that the filesystem cache is only covered by unrelated tests, and that the auto reload feature (timestamp) is not covered at all. Opened issues #1841 and #1842